src/libraries/System.Net.Security/src/System/Net/Security/SslStream.Implementation.cs

```diff
@@ -352,12 +352,12 @@ private async ValueTask<ProtocolToken> ReceiveBlobAsync<TIOAdapter>(TIOAdapter a
                 _framing = DetectFraming(_handshakeBuffer.ActiveReadOnlySpan);
             }
 
-            if (_framing == Framing.BeforeSSL3)
+            if (_framing == Framing.BeforeSSL3 || _framing == Framing.Unified)
```

Comment:
What does Unified mean? Can you add XML comments to the enum members to explain what they're used for?

Reply:
The `Unified` is transient state and perhaps not named very descriptively. It seems like in case of Ssl2, we wait for response and maybe upgrade the protocol and framing. So it is something like `MaybeSsl2`.  I added notes to the enum.